cset-workflow/meta/rose-meta.conf

```diff
@@ -115,697 +232,1353 @@ sort-key=met1
+help=The end date of the trial, in ISO 8601 format. This is the last date that
+    the workflow will run to. If blank, the workflow will run indefinitely.
+    For example: 2000-01-01T00:00Z
 type=quoted
 compulsory=true
+sort-key=c2
 
-[template variables=PLOT_RESOLUTION]
-ns=General
-description=Resolution of output plot in dpi.
-help=This is passed through to the plotting operators and sets the resolution
-    of the output plots to the given number of pixels per inch. If unset
-    defaults to 100 dpi. The plots are all 8 by 8 inches, so this corresponds
-    to 800 by 800 pixels.
-type=integer
+[template variables=CSET_TRIAL_CYCLE_PERIOD]
+ns=Models and Cases
+title=Trial cycle period
+description=The period between model runs of the trial.
+help=The cycle period of the trial, as an ISO 8601 duration. This is the time
+    between each forecast initiation of the model. For example: PT48H
+type=quoted
 compulsory=true
+sort-key=c3
 
-[template variables=WEB_DIR]
-ns=General
-description=Path to directory that is served by the webserver.
-help=This will probably be under $HOME/public_html or similar. You will want to
-    ensure a subfolder is used as multiple files will be written here.
+[template variables=CSET_ANALYSIS_PERIOD]
+ns=Models and Cases
+title=Analysis period
+description=The length of forecast to analyse, AKA forecast length.
+help=The period over which to produce diagnostics for the forecast, in ISO 8601 duration format.
+    For example: PT48H
+
+    This controls how much data is fetched and fed into CSET for analysis.
 type=quoted
 compulsory=true
-sort-key=web2
+sort-key=d1
 
-[template variables=WEB_ADDR]
-ns=General
-description=The address at which the website is served.
-help=This should be the address where your public_html or equivalent is served.
-    It might include a partial path, such as your username.
+[template variables=CSET_ANALYSIS_OFFSET]
+ns=Models and Cases
+title=Analysis offset
+description=Offset from forecast initiation to verification start.
+help=The offset in time between the forecast initiation and the start of the
+    analysis period. This is useful when needed fields are not output on the
+    first time step. For example: PT1H
 type=quoted
 compulsory=true
-sort-key=web1
+sort-key=d2
 
-[template variables=CSET_RUNAHEAD_LIMIT]
-ns=General
-description=Number of simultaneous cycles.
-help=The maximum number of cycles run in parallel. A larger number here will
-    finish quicker, but utilise more compute resources at once. For a large
-    enough workflow it may overwhelm the batch submission system, so it is
-    recommended to keep this below 10. Must be positive.
+[template variables=CSET_MODEL_COUNT]
+ns=Models and Cases
+title=Number of models
+description=Number of models to evaluate.
+help=The number of models to evaluate, between 1 and 10. If more models
+    are needed, it is straightforward to change upon request.
```

Comment:
Can users do this themselves or do they need to request this with us? Needs clarification in the text.

Reply:
Currently this requires a change to the rose-meta.conf, as well as one line in the jinja-utils script, so it would be a request to us. I couldn't come up with a reasonable use case for more than 10 models. If we implement Stephen's suggested meta generation script, then we could provide instructions for rerunning that.

Reply:
Moved to #817.